plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_aws_launcher.py

```diff
@@ -20,17 +17,13 @@ class RayAWSLauncher(Launcher):
     def __init__(
         self,
         mandatory_install: DictConfig,
-        ray_init_cfg: DictConfig,
-        ray_remote_cfg: DictConfig,
-        ray_cluster_cfg: RayClusterConf,
+        ray: RayAWSConf,
         stop_cluster: bool,
         sync_up: RsyncConf,
         sync_down: RsyncConf,
     ) -> None:
-        self.ray_init_cfg = ray_init_cfg
-        self.ray_remote_cfg = ray_remote_cfg
-        self.ray_cluster_cfg = ray_cluster_cfg
-        self.docker_enabled = self.ray_cluster_cfg.docker.image != ""
+        self.ray = ray
```

Comment:
ray sounds like the the ray module. can you assign to ray_cfg to make it clearer?